src/objects.js

```diff
@@ -1,41 +1,73 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+	name: name,
+	age: age,
+}
+return person
 };
 
 const getName = object => {
-  // your code here
+  return object.name
 };
 
 const getProperty = (property, object) => {
-  // your code here
+	const personDetails = object 
+  	return personDetails.age  
+
+/*I have no idea how the above can work but {const person = object 
+    return person.property} doesnt work? it should just substitute "age" into property but I get undefined each time?? I'm sure i'm just missing something obvious! */ 
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if(object.hasOwnProperty(property) === true){
+	return true}
+else {
```

Comment:
If you write it as return object.hasOwnProperty(property), you should get either true or false because that's what the function resolves to.